build.sbt

```diff
@@ -40,26 +28,11 @@ val root = project.in(file("."))
 
 lazy val core = project
   .enablePlugins(JavaServerAppPackaging)
-  .overrideConfigs(CustomMacro, CustomCompile, CustomRuntime, CustomTest, CustomIntegrationTest)
+  .compileMacros
   .settings(Defaults.itSettings)
```

Comment:
Is this still needed or can it be moved into Utils?  The CustomIntegrationTest config is specified as a part of the compileMacros method, so it should be safe to assume it's always available, I think.

It could also be unnecessary because of `inConfig(CustomIntegrationTest)(Defaults.testSettings)` in Utils.